bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -153,18 +171,10 @@ contract("Bridge NFT", async function(accounts) {
         const tokenURI = "/ipfs/QmYBX4nZfrHMPFUD9CJcq82Pexp8bpgtf89QBwRNtDQihS";
         let totalSupply = 0; // is the amount of nft minted
 
-        let receipt = await this.token.safeMint(tokenOwner, tokenId, {
-          from: tokenOwner,
-        });
-        utils.checkRcpt(receipt);
+        await mintAndApprove(this.token, this.bridgeNft.address);

```

Comment:
I'd maybe pass the tokenId to the `mintAndApprove` function, since we have one locally (now, it depends on the default value for tokenId being 9, this might break if someone new assumes stuff I guess)

Reply:
Agree, done at https://github.com/rsksmart/tokenbridge/pull/248/commits/5e908cbcb56c8924846eff80da10efa0adef6d0f